build.psm1

```diff
@@ -220,6 +220,10 @@ function Start-PSBuild {
         [switch]$NoPSModuleRestore,
```

Comment:
@rjmholt, your last commit had 2 failures in `PowerShell-CI-windows`
TestCustomPipeNameCreation
```powershell
Assert.True() Failure
Expected: True
Actual:   False
   at PSTests.Parallel.NamedPipeTests.TestCustomPipeNameCreation() in D:\a\1\s\test\xUnit\csharp\test_NamedPipe.cs:line 28
```
TestAppDomainProcessExitEvenHandlerNotLeaking
```powershell
Assert.DoesNotContain() Failure
Found:    (filter expression)
In value: Delegate[] [EventHandler { Method = Void DisposeOnShutdown(System.Object, System.EventArgs), Target = null }, EventHandler { Method = Void CurrentDomain_ProcessExit(System.Object, System.EventArgs), Target = null }]
   at PSTests.Sequential.RunspaceTests.TestAppDomainProcessExitEvenHandlerNotLeaking() in D:\a\1\s\test\xUnit\csharp\test_Runspace.cs:line 127
```